tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py27-django{18,111}
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE = eventtracking.django.tests.settings
+    PYTHONPATH = {toxinidir}
+
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/dev-requirements.txt
```

Comment:
It's a little weird that tox-requirements.txt is not actually used in tox. In other repos we deal with the shifting Django version dependency by having the expected Django version(s) in the main requirements file, then having a separate test-requirements file that has everything needed by the app + things for testing, minus Django. That's the file we would include here. This would also let you get tox-requirements.txt out of setup.py which also seems like a funny place for a file with that name.

Some examples of the pattern I mentioned are:
https://github.com/edx/help-tokens/blob/master/tox.ini
https://github.com/edx/django-user-tasks/blob/master/tox.ini